packages/UI/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -4,9 +4,12 @@ import Checkbox from './Checkbox';
 
 const meta: Meta<typeof Checkbox> = {
   component: Checkbox,
-  args: {
-    checked: false,
-    disabled: false,
+  argTypes: {
+    checked: { control: 'radio', options: [true, false, 'indeterminate'] },
+    defaultChecked: {
+      control: 'radio',
+      options: [true, false, 'indeterminate'],
+    },
```

Comment:
I added those to have better control on storybook's control tab when changing the props.

you can find the docs for changing the controls however you see fit.
https://storybook.js.org/docs/essentials/controls#annotation